### ISSUE TITLE
There exist several qeues in classic qeues version – this is exploited by the AI to build everything twice as fast

### ISSUE THREAD

**Issue by nmoskopp** [MlemandPurrs/raplusmod#9]
To reproduce: Queue a power plant from the interface in the sidebar, then click on your construction yard, then try to queue another power plant. Notice that players apparently can build two power plants at the same time.

```
Platform is Linux
Engine version is release-20210321
Using SDL 2 with OpenGL (Modern) renderer
Desktop resolution: 1920x1080
No custom resolution provided, using desktop resolution
Using resolution: 1920x1080
Using window scale 1,00
OpenGL renderer: Mesa Intel(R) UHD Graphics 620 (WHL GT2)
OpenGL version: 4.6 (Core Profile) Mesa 20.3.5
Using default sound device
Internal mods:
	modcontent: Mod Content Manager (release-20210321)
	raplus: Red Alert Plus (v.1.4.1-classicqueue)
External mods:
	sp-release-20220702: Shattered Paradise (release-20220702)
	ra-release-20210321: Red Alert (release-20210321)
	hv-20221030: OpenHV (20221030)
	hv-20220526: OpenHV (20220526)
	sa-20210523: OpenSA (20210523)
	raplus-v.1.4.1-multiqueue: Red Alert Plus (v.1.4.1-multiqueue)
	hv-20220918: OpenHV (20220918)
	hv-20221119: OpenHV (20221119)
	d2k-release-20210321: Dune 2000 (release-20210321)
	cnc-release-20210321: Tiberian Dawn (release-20210321)
	raplus-v.1.4.1-classicqueue: Red Alert Plus (v.1.4.1-classicqueue)
Loading mod: raplus
System.Net.Sockets.SocketOptionName 0x17 is not supported at IP level
[2022-11-26T17:35:17] Game started
[2022-11-26T17:35:54] Game started
AL lib: FreeContext: (0x1ad1c90) Deleting 32 Source(s)
```

**Comment by nmoskopp** [comment]
Reproduced with a single allied barracks.

**Comment by nmoskopp** [comment]
**It seems that the AI is abusing this bug to build twice as fast as the player.**

**Comment by nmoskopp** [comment]
This seems to be fixed in version 1.4.3.
